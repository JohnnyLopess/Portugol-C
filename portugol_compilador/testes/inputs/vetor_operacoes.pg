programa {
    funcao inicio() {
        inteiro a[3]
        inteiro b[3]
        a[0] = 10
        a[1] = 20
        b[0] = a[0] + 5
        b[1] = a[1] * 2
        escreva(b[0])
        escreva(b[1])
    }
}
