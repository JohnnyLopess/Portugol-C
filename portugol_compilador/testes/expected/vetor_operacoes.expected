#include <stdio.h>

int main() {
    int a[3];
    int b[3];
    a[0] = 10;
    a[1] = 20;
    b[0] = a[0] + 5;
    b[1] = a[1] * 2;
    printf("%d\n", b[0]);
    printf("%d\n", b[1]);
    return 0;
}
